resources/serverless/templates/tests/long-test.yaml

```diff
@@ -0,0 +1,85 @@
+{{- if .Values.tests.enabled }}
+{{- if .Capabilities.APIVersions.Has "testing.kyma-project.io/v1alpha1" }}
+apiVersion: "testing.kyma-project.io/v1alpha1"
+kind: TestDefinition
+metadata:
+  name: {{ include "fullname" . }}-long
+  labels:
+    app.kubernetes.io/name: {{ include "tests.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ include "chart" . }}
+spec:
+  disableConcurrency: {{ .Values.tests.disableConcurrency }}
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "true"
+    spec:
+      serviceAccountName:  {{ include "tests.fullname" . }}
+      restartPolicy: {{ .Values.tests.restartPolicy }}
+      initContainers:
+        - name: {{ include "tests.name" . }}-init
+          image: "{{ .Values.tests.long.image.repository  }}:{{ .Values.tests.long.image.tag }}"
+          imagePullPolicy: {{ .Values.tests.long.pullPolicy }}
+          command:
+            - "/bin/bash"
+          args:
+            - "-c"
+            - |
+              # create namespace and function for long running tests
+              (
+              cat <<EOF | kubectl apply -n {{ .Values.tests.long.namespace }} -f -
+              ---
+              apiVersion: v1
+              kind: Namespace
+              metadata:
+                name: {{ .Values.tests.long.namespace }}
+              ---
+              apiVersion: serverless.kyma-project.io/v1alpha1
+              kind: Function
+              metadata:
+                name: {{ .Values.tests.long.name }}
+              spec:
+                source: |
+                  module.exports = {
+                      main: function(event, context) {
+                        return 'OK'
+                      }
+                  }
+              ---
+              EOF
+              ) \
+              && kubectl wait --for=condition=Running functions.serverless.kyma-project.io/{{ .Values.tests.long.name }} -n {{ .Values.tests.long.namespace }}
+      containers:
+        - name: {{ include "tests.name" . }}
+          image: "{{ .Values.tests.long.image.repository  }}:{{ .Values.tests.long.image.tag }}"
+          imagePullPolicy: {{ .Values.tests.long.pullPolicy }}
+          command:
+            - "/bin/bash"
+          args:
+            - "-c"
+            - |
+              echo 'TESTING has started'
+              sleep 20
+
+              # test start
+
+              curl -XGET http://{{ .Values.tests.long.name }}.{{ .Values.tests.long.namespace }} \
```

Comment:
Please also check if Function CR has all conditions set to `true`, as status can be invalid.

Reply:
Please also check that response is `OK` :)

Reply:
@magicmatatjahu The result code is enough, why you want to check the response body?